src/nft/components/details/detailsV2/DataPage.test.tsx

```diff
@@ -3,7 +3,12 @@ import { render } from 'test-utils/render'
 
 import { DataPage } from './DataPage'
 
-it('placeholder containers load', () => {
-  const { asFragment } = render(<DataPage asset={TEST_NFT_ASSET} />)
+it('data page loads with header showing', () => {
+  const { asFragment } = render(<DataPage asset={TEST_NFT_ASSET} showDataHeader={true} />)
+  expect(asFragment()).toMatchSnapshot()
+})
+
+it('data page loads without header showing', () => {
```

Comment:
You're not actually asserting these expectations. You should include some sort of selector into the rendered component ensuring the header is/is not shown, _along_ with the snapshot.

The snapshot acts as a guard that nothing changes, but if you do change something, and it breaks the header, and a dev just blindly updates the snapshot, you won't catch the breakage.

Reply:
With the header we still need its spacing so to hide it we're setting `opacity: 0` thus the element is still in the DOM so we can't simply check if it's present/not present. Jest's method `.not.toBeVisible()` also doesn't work with opacity, instead it relies on `display: none`. Jest style checking only works for inline styles so `.toHaveStyle('opacity: 0')` never returns true since we are using styled components.

Reply:
This is definitely not ideal but sounds fine for this use case. You may want to leave a comment in the file explaining why you were unable to do more than just snapshot tests.

Reply:
Added a comment to provide context